server/lib/matrix-utils/timestamp-to-event.js

```diff
@@ -18,7 +18,7 @@ async function timestampToEvent({ accessToken, roomId, ts, direction }) {
 
   const timestampToEventEndpoint = urlJoin(
     matrixServerUrl,
-    `_matrix/client/unstable/org.matrix.msc3030/rooms/${encodeURIComponent(
+    `_matrix/client/v1/rooms/${encodeURIComponent(
```

Comment:
As a heads-up, this PR will probably wait until the new year.

Reply:
> Any update on when this will be merged?

Maybe a couple weeks to a month out. There are extenuating circumstances behind the scenes (not that ominous)

Reply:
This is now resolved 🎉